pkg/shp/cmd/buildrun/list.go

```diff
@@ -91,11 +93,30 @@ func (c *ListCommand) Run(params *params.Params, io *genericclioptions.IOStreams
 	}
 
 	if !c.noHeader {
+		if c.wide {
+			columnNames = "NAME\tSTATUS\tAGE\tSOURCE\tOUTPUT-IMAGE\tBUILD-NAME\tELAPSED-TIME\tSOURCE-ORIGIN"
```

Comment:
I'm OK with this as a "first pass" on this feature.

Something to think about is column widths - a simple tab formatting can cause the headers to be too short for the respective row content. There might be a cli "table" library that the kubernetes `cli-runtime` uses and we can take advantage of (see https://github.com/shipwright-io/cli/blob/main/go.mod#L16).